src/refactor_nrepl/ns/clean_ns.clj

```diff
@@ -36,13 +36,13 @@
   (assert-no-exclude-clause (core/get-ns-component ns-form :use))
   ns-form)
 
-(defn- find
+(defn- seek
   "Find the first element in coll that satisfies pred?"
   [pred? coll]
   (reduce (fn [_ x] (when (pred? x) (reduced x))) nil coll))
 
 (defn clean-ns [{:keys [path relative-path]}]
-  (let [path (find #(and % (.exists (io/file %)))
+  (let [path (seek #(and % (.exists (io/file %)))
```

Comment:
Just use `fn` instead of the `#()` notation with `some`.  

I agree with you that it's awkward to continue expanding the `#()` but implementing `seek` as a slight variation of `some` isn't any better.  I just shifts the question from "What does this hard-to-read anonymous function do" to "what does `seek` do"?

Reply:
Sure, whatever you prefer. This was prompted by some discussion on twitter, the reduce/reduced version has some benefits in terms of performance, but I agree it's overkill in this case.

https://mobile.twitter.com/plexus/status/1119957160685985793